app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        try:
+            copy, old_file, new_file = command.split()
```

Comment:
Don't call `split` twice. You should create variable for storing result of this function and use everywhere you need